src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -47,13 +47,24 @@ websocket.filter.label.regex.inverse            = Inverse
 websocket.fuzz.fail                             = Error
 websocket.fuzz.success                          = Successful
 websocket.manual_send.fail                      = Unable to send crafted message!
+websocket.manual_send.fail.unable_reopen        = It is not able to re-open Connection.
+websocket.manuual.send.fail.non_supported_ver   = This WebSocket Version haven't been supported yet.
 websocket.manual_send.fail.disconnected_channel = Selected WebSocket channel is not connected.
 websocket.manual_send.fail.invalid_channel      = Invalid WebSocket channel selected.
 websocket.manual_send.fail.invalid_opcode       = Invalid WebSocket opcode selected.
+websocket.manual_send.use_reopen                = Use the reopen button to establish a new connection.
```

Comment:
Reopen needs to be consistent between all exceptions above and these values, either `reopen` or `re-open`.